packages/core/src/core/contentGeneratorConfig.test.ts

```
@@ -0,0 +1,101 @@
+      process.env.OPENAI_API_KEY = 'test-key';
+
+      const config = await createContentGeneratorConfig(
+        'gpt-4-turbo',
+        AuthType.USE_OPENAI,
+      );
+
+      expect(config.model).toBe('gpt-4-turbo');
+    });
+
+    it('should not return config without API key', async () => {
+      delete process.env.OPENAI_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)